Pear/Controller/HomeViewController.swift

```diff
@@ -85,6 +87,31 @@ class HomeViewController: UIViewController {
         navigationController?.setNavigationBarHidden(false, animated: false)
     }
 
+    private func showInAppFeedback() {
+        guard let netId = UserDefaults.standard.string(forKey: Constants.UserDefaults.userNetId) else { return }
+        NetworkManager.shared.getMatchHistory(netID: netId).observe { response in
+            switch response {
+            case .value(let currentMatchHistory):
+                guard currentMatchHistory.success else {
+                    print("Network error: could not get user match history")
+                    return
+                }
+                var previousMatchHistorySize = UserDefaults.standard.integer(forKey: Constants.UserDefaults.previousMatchHistorySize)
+                // the default value for previousMatchHistorySize is 0, but it should be 1 since the app feedback form should only be shown when the user has a match history of size 2 or more
```

Comment:
I'm actually still a little confused about this haha. Why should `previousMatchHistorySize` be 1? We show app feedback form when user has match history of size > 2. So regardless of if `previousMatchHistorySize` is 0 or 1, the check should still be the same?